db/c.cc

```diff
@@ -3156,6 +3420,31 @@ void rocksdb_options_set_merge_operator(
   opt->rep.merge_operator = std::shared_ptr<MergeOperator>(merge_operator);
 }
 
+rocksdb_table_properties_collector_t* rocksdb_table_properties_collector_create(
```

Comment:
This function is used by the `create_collector` callback passed to `rocksdb_options_add_table_properties_collector_factory` below. We don't expose a corresponding `rocksdb_table_properties_collector_destroy` as the returned struct is owned by RocksDB.